actions/transactions/data_get_test.go

```diff
@@ -0,0 +1,58 @@
+package transactions_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/bitcoin-sv/spv-wallet/actions/testabilities"
+	"github.com/bitcoin-sv/spv-wallet/actions/testabilities/apierror"
+	"github.com/bitcoin-sv/spv-wallet/engine/tester/fixtures"
+)
+
+//NOTE: Standard Flow Case is tested in outlines_record_test.go
+
+func TestGetTransactionDataErrorCases(t *testing.T) {
+	t.Run("no data under outpoint", func(t *testing.T) {
+		// given:
+		given, then := testabilities.New(t)
+		cleanup := given.StartedSPVWallet()
+		defer cleanup()
+		client := given.HttpClient().ForGivenUser(fixtures.Sender)
+
+		// when:
+		res, _ := client.R().Get(fmt.Sprintf("%s/%s/0", getTransactionDataURL, fixtures.GivenTX(t).ID()))
+
+		// then:
+		then.Response(res).
+			HasStatus(404).
+			WithJSONf(apierror.ExpectedJSON("error-transaction-data-outpoint-not-found", "data outpoint not found"))
+	})
+
+	t.Run("try to get as admin", func(t *testing.T) {
+		// given:
+		given, then := testabilities.New(t)
+		cleanup := given.StartedSPVWallet()
+		defer cleanup()
+		client := given.HttpClient().ForAdmin()
+
+		// when:
+		res, _ := client.R().Get(fmt.Sprintf("%s/%s/0", getTransactionDataURL, fixtures.GivenTX(t).ID()))
```

Comment:
One minor question, why the second value is omitted? 🤔

Reply:
Hmmm I've got this pattern from @dorzepowski 🙈 
- So I don't know either 🤷

Reply:
Ehhh. I didn't thought it will be soooo misleading.
The client provided from testabilities is configured such a way, if there is any error (by error I mean programatic error, **not** response with status 4xx or 5xx) the test will fail. So there is no need to check that one because in case of error the line checking the error will never be called.